demo_fastapi/sales.py

```diff
@@ -31,7 +47,7 @@ def create_order(
         )
 
     # check date format and if it's after 1 January 2000
-    r = re.compile("\d{4}/\d{2}/\d{2}")
+    r = re.compile(r"\d{4}/\d{2}/\d{2}")
```

Comment:
Doesn't this test belong to the model? Couldn't we handle the test on the model side and handle the exceptions here to send back meaningful responses to the use here?
ps: just a suggestion, we could talk about this a bit more if you want.

Reply:
The problem was that checking types and formatting in pydantic model did not give me meaningful responses when I load app docs. Only HTTP exceptions printed something meaningful.
I am not sure how I would inform the app which validation in the model failed and how the app could return the same error message but through HTTP exception. Feel free to give me advice, I am a noob at this! :smile: